graph-db/serverless.yml

```diff
@@ -13,15 +13,15 @@ provider:
    name: graphdb-driver-deployment-bucket-${self:provider.stage}
    skipPolicySetup: true
 
-  vpc:
-    securityGroupIds:
-      - ${cf:${self:provider.stage}.servicesSecurityGroupId}
-    subnetIds: !Split [',', '${cf:${self:provider.stage}.subnetIds}']
+  # vpc:
```

Comment:
why are these commented out, are they needed for an environment that you use in development? Is it code that will be implemented soon? Basically I want to know why it is not just deleted, no comment was given

Reply:
I have since restored this in the latest commit; in the test session for the related ticket I had added commenting this out again as part of the test session as this being tested locally.